src/test/java/com/karankumar/bookproject/backend/utils/PredefinedShelfUtilsTest.java

```diff
@@ -49,35 +48,34 @@
 
 @IntegrationTest
 class PredefinedShelfUtilsTest {
-    private static BookRepository bookRepository;
+    @Autowired private BookRepository bookRepository;
+    @Autowired private PredefinedShelfService predefinedShelfService;
```

Comment:
I agree that we shouldn't have had these in setUp(). I was going to fix this later, so thanks for this. Could you please autowire these in via constructor injection?